test/page.spec.ts

```diff
@@ -1639,6 +1640,50 @@ describe('Page', function () {
dNJqN975R1lB4f6GzuEplB/sRdd303FY8l3pWTIH8/J1EuiadHDoyzP99889+Zn//J+TfZAboczgfqFar4WZBI4T9nzje1mlT4vdD+D/x+Vv99B7OSJ9jz/H2e93pfC0bpumiUHctW5pj1O9/Bd12YjTzJ3/X6B+8t5XYtZTa4l2G7dBVvV8Ts3bve4a1nz0CZzNqburBAYZUyQ6VSfxrvzapcHj7YI/5c6BMd1d8Nhucy/bkG7bnkH31uKbd72XBuN++tNlZyGz1A/w++LY35thlBxPec9B1/KOUvqeFJbafHD+vRkdLOW3bsuOU/TuzYectO6B3GiL7K34WCnpHVuu0xmEslvaxaNVqKRa3O2qjXQ9sX+lkGrYBCETV31zZfd6a/zszfmZp8eO/Dk/v5n2YvmgAfcrym75DuR2MlcoOkpLCqtFE4O2f4tNcbXWABCW1CCxgqqcTAozAmIz/hL6kmrP0tcvd7EdJQHsPwFrnnHV2EL3MGcSlrnmhTGQwCf+Gh/vpDUkyaAkkxXc8HcK+6qWwwlIfiT1wfKjocwUzL5htCNOSo/cZkpE7HHwM3b/bnbXZP++BN34hSNtCcGDx6l0kQdtouXTuYzA6snzAZhB3y3ce1OFOAWoWreA1ElNys6ZlqkUraUAQQMC9TJvjuldld1Gyiic9kGEvb36swAl9wBZHPRDXUi0rI+GfcGurFeYySV3spHlm1YjG/263IlQWlFHTOYnrSAr77nE0HhAD9wH1rqCE9fAD8LfHAitRqH2W7l+xm1MduHhqqfYJeC5+v/6Yl6P5NbRUV4z/6Ubx2js8pbs7Rc6BL2XolEyUL37gUDe94Ma9DTsp4S00k/x/BSiRPAAAAeNpjYGRgYADiK5MK/8Xz23xlkGd+ARRhOPfc3xhG/z/5X495CnMRkMvBwAQSBQCRpQ36AHjaY2BkYGCu+S8FJF/8PwkkpzAARVBAKwB9OgVqAAAAeNo1kTtM02EUxX/3/APKYuiCMTQK/nmoSKVpWjQoaAgmao3iUFQS2vAwDj6AkGjcXB3VzTgww8Dg4mbcjImDLiS64aCzj9DEgKcow8m537333NcXd+hMvkEDEcYwd6PCiD5wXCVKqjOgVUq8ZyRyXDCy8YkurXCZbdpjnZtsUY7n9i045zsFnWZQgxzRFIdUsH6CIWVIVeWU7Xx0UY1uTihHp15SMw9Z15EUnbfEXj2y76N1X81njBZjk7x+UovU9gEO6r45Qy2Z5qJ+mYvOr5rPOe+xY4c5pnGatE5FD2lOXrnXLG0qu36Vo9HPnGfOep9Uc4zpmu0fLKvV7+uUlXr+G/Spz7lXbY/bnqRMnUtsbr/TPu/cwvnkqXPt1zy9Ozpr4oFjb0jjlvevMBa/2ZPkyMQf2mILqYnemPId2pmJxo1qjDZur7MUdYWTni/rfUqN2+6w54rX/qNF/88u9wBfjGf/cc9YM/KOWbcLvXWNJ/6jDUc+U4gF971tfkEhkfdZ+1fHfXuSVfZHHf4CxQ1cPgB42mNgYNCCwjiGCYxpjH+YpjHrMPsx5zH3MR9i/sciw2LFEseSw7KG5QVrCOsy1h9sdWwf2OXYl3FIcLhw5HE0cSzi2MYpwtnGuYtLjiuLawnXPW4h7h7uI9yPeDR4EnjaeF7wWvE28B7iU+Jr4PvEr8Qfw9/G/01ATyBJYInAJYEHgkKCeoIegmmC94Q8hKYJMwinCG8S0REpE1kkckdURNRGdJ3oLzEHsU3iMuIR4sckRCTiJPokNSTjJOdIXpPykkqTuiatBIQt0k9kvGQeyLyRnSS7AQfcI3tC9orsGzkWOSU5B7kcuRlyB+T+gKB8ifwzBTYApapNfgAAAQAAAIUAOwAFAAAAAAACAAEAAgAWAAABAAFJAAAAAHjapZG9TgJBEMf/C4iChmBiDJVuic15QDCEwsSPSkMjJtR3eCCCd4Y7JFb3ChYWlsb4MPoSPoSVpf/dWwyJYuNNbue387E7MwtgUxQgoL75NaV1wmlNCWew9c1Z6m16RSbH3Q2kYcGoe8Mpeh4Mp+c4Q5nxEvJ4NJxlxrPhZZzjxfAKSvg0nENJFA3n0RRlw6vYF67hNfKT4SI2xLvhdWTFh+FX9j478w12qhC3JqEXyLbjh7Ju2/LM609Gzlh2PLcX+FGMIwTs8w5jDNDHJSL2XEYXO9RV2JQGyWWExAljQ+pDTBh9Cw8+cxxYtB1gRJFz54R651F71Cr6gpEWYv1XuNpaYrR4nooKmNHmeb7OHejV4R8x32G2h2vqMYa0Bej9WdEizyJ7m1Nw2LeqUuoalB7q2n1jnTIjYm+qtlPjsXSvEbOb2KVMtVi40ve4P+pKugh0ps+J/S/7t8nV9VQ7tLqckYpVb5rMu6X3Esf0dqlrtClfhfdXsafXhnn5CmpfaA13OgAAeNptz1cvg3EUx/HvoUMHWkrtvefzlC67Ouy9NwlaiSCVXrgnEuPWm3CH8vJo2v+lc/PJ75zknBzyyNZvGg//1QNInuSTjwEjJswUYMGKDTuFFFGMAycllOKijHLcVFBJFdXUUEsd9TTQSBPNtNBKG+100EkX3fTQSx/9aOiZ2wMM4sWHnwBBhhhmhFHGGGeCEJOEiRAlxhTTzDDLHPMssMgSy6ywyhrrbLDJFtvssMse+xxwyBHHnIiB+8wnbzyLkSdexSRmKRCLWMUmdimUIikWhzilREr54JNvfvgizSPv4pIyKedF3Kb45d1NQjenri40TYvkDGnKbPZkBkpd6VEOKAeVXqVP6VcGlEFlKKeu9uq69fwinkqenZ7cJnItTyynN2aIppLX2eCNhf8AwzpP+gB42tvB+L91A2Mvg/cGjoCIjYyMfZEb3di0IxQ3CER6bxAJAjIaImU3sGnHRDBsYFZw3cCs7bKBTcF1E7MDkzaYwwrksGlCOSxADqsslMMOUpYH4TBu4IBq5lRw3cXAUf+fgUl7I7NbmbZL5AYRbQDhkycvAAABUsGetAAA';
```

Comment:
@nikli2009 Is this only happening for base64 decoded fonts or could this also be an url?

Reply:
🤔 good question, gonna try some other combination / scenarios to see if it could break the pdf call